__tests__/integration.test.js

```diff
@@ -66,6 +66,7 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           article_img_url:
             "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+            comment_count: 11
```

Comment:
This should be a new test - it is new behaviour. Updating tests for new behaiviour is not TDD. If you _had_ to update this test then it means the test was "brittle". Ideally it should have been originally written in a way that asserted correctly but didn't fail for added behaviours.